hack/perf/bz-2162111.sh

```diff
@@ -21,23 +21,31 @@ container_cmd="--name $container_name \
 	-v $file_b:/home/file_b:rw \
 	--cap-drop=ALL \
 	--ulimit nofile=1024:2048 \
-	$IMAGE true"
+	$IMAGE"
 
 # Script to clean up before each benchmark below
 prepare_sh=$(mktemp -p $tmp --suffix '.prepare.sh')
 cat >$prepare_sh <<EOF
-\$ENGINE rm -f \$(\$ENGINE ps -aq)  > /dev/null
-\$ENGINE volume prune -f            > /dev/null
-\$ENGINE volume create $volume_name > /dev/null
+\$ENGINE rm -f \$(\$ENGINE ps -aq)
+\$ENGINE volume prune -f
+\$ENGINE volume create $volume_name
 EOF
 echo_bold "Prepare script: $prepare_sh"
 
 # Script to create container below
 create_sh=$(mktemp -p $tmp --suffix '.create.sh')
 cat >$create_sh <<EOF
-\$ENGINE create $container_cmd > /dev/null
+\$ENGINE create $container_cmd true
 EOF
-echo_bold "Create script: $prepare_sh"
+echo_bold "Create script: $create_sh"
+
+# Script to run container below
+run_sh=$(mktemp -p $tmp --suffix '.run.sh')
+cat >$run_sh <<EOF
+\$ENGINE rm -f $container_name || true
```

Comment:
Is the `|| true` a knock-on-wood in case the container doesn't exist? I don't think that's needed any more. Or is there another more insidious reason for it? (Not worth a repush for this only; but worth removing or commenting if there's another reason to repush).

Reply:
> Is the || true a knock-on-wood in case the container doesn't exist?

Yes.  The first run would otherwise fail.  Podman as an `--ignore` flag but Docker does not, so I went with with `|| true` cheat.

Reply:
I think `-f` now implicitly succeeds on nonexistent containers, and has for a while?
```console
$ bin/podman rm -f dsfsdfsdf && echo ok
ok
```

Reply:
> I think -f now implicitly succeeds on nonexistent containers, and has for a while?

I forgot about that, indeed.  Docker would still error out :(